apps/protoc-gen-openapi/examples/tests/jsonoptions/openapi.json.yaml

```diff
@@ -0,0 +1,85 @@
+# Generated with protoc-gen-openapi
+# https://github.com/google/gnostic/tree/master/apps/protoc-gen-openapi
+
+openapi: 3.0.3
+info:
+    title: Messaging API
+    description: Messaging service
+    version: 1.2.3
+paths:
+    /v1/messages/{messageId}:
+        post:
+            tags:
+                - Messaging
+            operationId: Messaging_CreateMessage
+            parameters:
+                - name: messageId
+                  in: path
+                  required: true
+                  schema:
+                    type: string
+                - name: notUsed
+                  in: query
+                  schema:
+                    type: string
+            requestBody:
+                content:
+                    application/json:
+                        schema:
+                            type: string
+                required: true
+            responses:
+                "200":
+                    description: OK
+                    content:
+                        application/json:
+                            schema:
+                                $ref: '#/components/schemas/message'
+    /v1/messages/{message_id}:
```

Comment:
I didn't know about Spectral - I will check that out.
As far as I remember, I named them differently to ensure that it would work with mixed naming conventions, but I can just change it, if that is preferred?

Reply:
No worries, I don't think this needs to be changed.